src/inventory_client/inventory_client.go

```diff
@@ -270,10 +270,12 @@ func (c *inventoryClient) ListsHostsForRole(ctx context.Context, role string) (m
 	return ret.Payload, nil
 }
 
-func (c *inventoryClient) getMonitoredOperators(ctx context.Context, clusterId string) (models.MonitoredOperatorsList, error) {
+func (c *inventoryClient) getMonitoredOperators(ctx context.Context, clusterId string, useCache bool) (models.MonitoredOperatorsList, error) {
 	cacheKey := fmt.Sprintf("getMonitoredOperators-%s", clusterId)
-	if val, err := c.cache.Get(cacheKey); err == nil {
-		return val.(models.MonitoredOperatorsList), nil
+	if useCache {
```

Comment:
We could reduce levels a bit by making this part of the if condition on line 276